src/toasts.ts

```diff
@@ -94,7 +94,7 @@ export class Toast {
     // Create new toast
     Toast._toasts.push(this);
     const toastElement = this._createToast();
-    (toastElement as any).M_Toast = this;
+    toastElement['M_Toast'] = this;
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition